datadog/resource_datadog_dashboard_json.go

```diff
@@ -176,3 +240,38 @@ func updateDashboardJSONState(d *schema.ResourceData, dashboard map[string]inter
 	}
 	return nil
 }
+
+func updateDashboardJSONLists(d *schema.ResourceData, providerConf *ProviderConfiguration, dashboardID, layoutType string) {
```

Comment:
Actually, can't you reuse directly the same function as in the Dashboard resource ? Seems very similar apart from that `layoutType` argument